catalog/ui/search-ui/search-ui-app/src/main/resources/features.xml

```diff
@@ -36,6 +36,8 @@
              description="The DDF Standard Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.\nThe left pane of the SearchUI contains basic fields to query the Catalog and other Sources. The right pane consists of a map.\nIt includes both standard (3d globe) and simple (text page) versions.::DDF Standard Search UI">
         <configfile finalname="/etc/org.codice.ddf.ui.searchui.filter.RedirectServlet.config" override="false">mvn:ddf.ui.search/search-ui-app/${project.version}/config/redirect</configfile>
         <feature prerequisite="true">catalog-app</feature>
+        <feature prerequisite="true">catalog-versioning-plugin</feature>
```

Comment:
I would not have expected changes to the search ui features file other than to merge in the catalog ui features file into it.